packages/requisitions/src/RequestRequisition/DetailView/RequestLineEdit/ItemCharts/StockEvolution.tsx

```diff
@@ -31,17 +35,20 @@ export const StockEvolution: React.FC<StockEvolutionProps> = ({ id }) => {
     switch (name) {
       case 'stockOnHand':
         return [value, t('label.stock-level')];
-      case 'min':
+      case 'minimumStockOnHand':
         return [value, t('label.min')];
-      case 'max':
+      case 'maximumStockOnHand':
         return [value, t('label.max')];
       default:
         return [value, name];
     }
   };
-
   if (!data || !data.stockEvolution) return null;
   const tooltipLabelFormatter = (date: string) => dateFormatter(date);
+  const chartData = data.stockEvolution.nodes.map(node => ({
```

Comment:
was hoping to avoid this.. but, it does make the dynamic display of the chart pretty neat, when you adjust requested quantity